src/components/shortcuts.js

```diff
@@ -165,6 +166,14 @@ function showPlayMenu(card, target) {
     });
 }
 
+function getSortValues(parentId) {
+    const basekey = 'items-' + parentId + '-Folder';
+    return {
+        sortBy: userSettings.getFilter(basekey + '-sortby'),
+        sortOrder: userSettings.getFilter(basekey + '-sortorder') === 'Descending' ? 'Descending' : 'Ascending'
+    };
+}
```

Comment:
This is probably a candidate for extraction into some utils module (`utils/`) - it is also used by the list.

Reply:
I agree. I'm new to the code so I had no idea about the existence of utils modules.

Will port the code there.

Thanks!

Reply:
I think we also can move this to `userSettings` as `getSortValues(basekey)` and `setSortValues(basekey)` - just to limit the use of those magic suffixes.